### DISCUSSION TITLE
Use Wikibase or Wikidata itself to store data

### DISCUSSION THREAD

**Discussion by lorenzoMazza** [tosdr/edit.tosdr.org#999] [Ideas]
I think the problems with the actual system are:
* cases have to be organized in a hierarchy, not only under topics, in order to make them easier to find;
* some cases are not so meaningful (what does it mean that the scope of the copyright license is [very broad](https://edit.tosdr.org/cases/133)?).

If we'll use Wikibase (or Wikidata) to manage data - treating services, documents and cases as items and topics as properties - it would allow to:
* solve the first problem;
* contribute easier and quicker;
* in the case of Wikidata, a lot of [tools](https://www.wikidata.org/wiki/Wikidata:Tools) like SPARQL query.

**Comment by ghost** [comment]
(those interested in this issue may also wish to express their thoughts at https://forum.tosdr.org/t/wikidata-linking/53)

As a Wikidata enthusiast I am certainly interested in seeing the linked data paradigm be considered with respect to the useful database ToS;DR is building, but I do think some challenges have to be kept in mind. First, as you say the cases and topics are essentially a folksonomy, so there's questions about Wikidata's notability rules (which are much less strict than Wikipedia's, Wikimedia Commons', or the other sister projects, but are still there). ToS;DR's existence as an external ID ameliorates that, but if it delegates the case/topic structure to Wikidata wholesale, then that quality is somewhat lost.

Notability also applies to the stored documents; while a policy in effect is probably notable if the service it describes is (and has some precedent for journals' / universities' open-access policies, which was a Wikiproject Open Access import), what about superceded policies? ToS;DR and ToSback keep changelogs, but I think in Wikidata these would have to be either separate items or all the statements would have some complex start- and end-time qualifiers attached. And not all services are notable, plenty of people want to display a ToS;DR rating badge on the policies for their personal sites and currently do so. Finally, we have to respect Wikidata's right to curate their database, meaning that identifiers that in a naive cooperation scheme ToS;DR would depend on may end up deleted, redirected, or repurposed (although that last one is discouraged), breaking the chain of verification that ToS;DR currently provides.

None of this is to say that benefits might not be reaped, just that thought has to be put into doing so.
